docs/blog/posts/announcing-keptn-v2-release-candidate.md

```diff
@@ -0,0 +1,148 @@
+---
+date: 2024-03-13
+authors: [agardnerit,bacherfl]
+description: >
+  Announcing the Official Release Candidate for Keptn v2
+categories:
+  - Installation
+  - Upgrade
+comments: true
+---
+
+# Announcing the Official Release Candidate for Keptn v2
+
+The Keptn project is proud to announce a release candidate for what will become Keptn v2.
+This release candidate provides the project with enough end-user validation
+and bug-finding so that we can release Keptn v2.
+We anticipate (as long as no release blockers are identified) that Keptn
+v2 will be released approximately 1 month after this announcement.
+
+<!-- more -->
+
+## What Problem Does Keptn Solve?
+
+Deploying software using Kubernetes is, on the surface, easy.
+Just use `kubectl` or a GitOps solution like ArgoCD to deploy a YAML file and that's it, right?
+Well, no.
+Not normally.
+There is a lot more to ensuring a healthy deployment,
+and there are a lot of potential problems that can
+prevent the successful deployment of a Kubernetes application, such as:
+
+- Not enough resources (i.e. CPU, Memory) being available in the cluster.
+- An external dependency (e.g. a Database hosted outside the cluster) not being reachable.
+- Unhealthy state of your infrastructure (e.g. due to vulnerabilities).
+- The cluster or namespace not being monitored correctly - You would not want to deploy into the dark in this case.
+- Unexpected performance degradation after rolling out a new version of your application.
```

Comment:
```suggestion
- Not enough resources (such as CPUs or memory) are available in the cluster
- An external dependency (such as a database that is hosted outside the cluster) is not reachable
- The state of your infrastructure is unhealthy because of vulnerabilities
- The cluster or namespace is not monitored correctly; you do not want to deploy into the dark in this case
- Unexpected performance degradation after rolling out a new version of your application
```